smarty/tests/test_sampler.py

```diff
@@ -57,14 +56,14 @@ def test_atomtyper_elemental(self):
         """
         Test elemental atomtype sampler for hydrogen
         """
-        atomtype_sampler = smarty.AtomTypeSamplerElemental(self.mols_alkethoh,
+        atomtype_sampler = smarty.AtomTypeSampler(self.mols_alkethoh,
                 self.basetypes, self.basetypes, self.combine_decs,
                 self.replacements, self.mols_alkethoh_ref, 0.1, False,
-                'combinatorial-decorators', '1')
+                'combinatorial-decorators', 1)
```

Comment:
Ah, I see. OK, stylistically, I find it easier to read what's going on if you say `element=` for an optional argument... But, if that complicates too much you can leave it this way. Your call.